components/camel-telegram/src/main/java/org/apache/camel/component/telegram/model/InstantSerializer.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.telegram.model;
+
+import java.io.IOException;
+import java.time.Instant;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+/**
+ * A serializer for {@link Instant} compatible with {@link UnixTimestampDeserializer}.
+ */
+public class InstantSerializer extends JsonSerializer<Instant> {
```

Comment:
For my information, isn't this case covered by any of the jackson modules ?

Reply:
Not sure which other Jackson modules you mean? 

Some test code now serializes IncomingMessages (which was not the case before this PR) and InstantSerializer is just a serializer symmetric to UnixTimestampDeserializer so that for an IncomingMessage m the following pseudo-expression is always true: serialize(deserialize(m)).equals(m)

UnixTimestampDeserializer lives here so I thought InstantSerializer can also live in the same package.

Reply:
Jackson provides a module for java8 bits (https://github.com/FasterXML/jackson-modules-java8) that includes support for Instant & co. Not sure if it formats the Instant as expected here but worth investigating

Reply:
Searched in jackson-datatype-jsr310 and haven not found anything useful. The Telegram API is using epoch seconds without a point and fraction digits which seems to be non-standard.

Reply:
ok so fine to have our own impl, maybe we should just name the classes consistently

Reply:
Done in 733c862